moveit_ros/planning/planning_pipeline/src/planning_pipeline.cpp

```diff
@@ -277,8 +278,11 @@ bool planning_pipeline::PlanningPipeline::generatePlan(const planning_scene::Pla
       m.action = visualization_msgs::Marker::DELETEALL;
       arr.markers.push_back(m);
 
+      auto path_constraints_resolved = req.path_constraints;
+      kinematic_constraints::resolveConstraintFrames(res.trajectory_->getFirstWayPoint(), path_constraints_resolved);
+
       std::vector<std::size_t> index;
-      if (!planning_scene->isPathValid(*res.trajectory_, req.path_constraints, req.group_name, false, &index))
+      if (!planning_scene->isPathValid(*res.trajectory_, path_constraints_resolved, req.group_name, false, &index))
```

Comment:
I think line [325](https://github.com/moveit/moveit/blob/81822f2cd660739c10e11b6bee9e19448da0dd28/moveit_ros/planning/planning_pipeline/src/planning_pipeline.cpp#L325C15-L325C101) should also use the `path_constraints_resolved` :
```cpp
              // check validity with verbose on
              const moveit::core::RobotState& robot_state = res.trajectory_->getWayPoint(it);
              planning_scene->isStateValid(robot_state, req.path_constraints, req.group_name, true);
```